File: Modul 4 Sem 1/PHYTHON/PRAK404-2410817110005-MuhammadRizqanKarim.py
while True:
    print("\nPilih program:")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    
    pilihan = int(input("Masukkan pilihan: "))
    
    if pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator Muhammad Rizqan Karim")
        break
    elif pilihan < 1 or pilihan > 5:
        print("Input anda salah, silahkan coba lagi")
        continue

    nilai_pertama = float(input("Masukkan nilai pertama: "))
    nilai_kedua = float(input("Masukkan nilai kedua: "))
    
    if pilihan == 1:
        hasil = nilai_pertama + nilai_kedua
        operasi = "Penjumlahan"
    elif pilihan == 2:
        hasil = nilai_pertama - nilai_kedua
        operasi = "Pengurangan"
    elif pilihan == 3:
        hasil = nilai_pertama * nilai_kedua
        operasi = "Perkalian"
    elif pilihan == 4:
        if nilai_kedua == 0:
            print("Pembagian dengan 0 tidak dapat dilakukan.")
            continue
        hasil = nilai_pertama / nilai_kedua
        operasi = "Pembagian"
    
    print(f"Hasil {operasi} antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")

File: Modul 4 Sem 1/C/PRAK404-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>

int main() {
    int pilihan;
    float nilai_pertama, nilai_kedua, hasil;

    do {
        printf("\nPilih program:\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Muhammad Rizqan Karim\n");
            break;
        } else if (pilihan < 1 || pilihan > 5) {
            printf("Input anda salah, silahkan coba lagi\n");
            continue;
        }

        printf("Masukkan nilai pertama: ");
        scanf("%f", &nilai_pertama);
        printf("Masukkan nilai kedua: ");
        scanf("%f", &nilai_kedua);

        switch (pilihan) {
            case 1:
                hasil = nilai_pertama + nilai_kedua;
                printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                break;
            case 2:
                hasil = nilai_pertama - nilai_kedua;
                printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                break;
            case 3:
                hasil = nilai_pertama * nilai_kedua;
                printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                break;
            case 4:
                if (nilai_kedua == 0) {
                    printf("Pembagian dengan 0 tidak dapat dilakukan.\n");
                } else {
                    hasil = nilai_pertama / nilai_kedua;
                    printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                }
                break;
        }
    } while (1);

    return 0;
}
